cdap-docs/reference-manual/source/http-restful-api/artifact.rst

```diff
@@ -197,6 +197,191 @@ artifact (pretty-printed and reformatted to fit):
       "version": "|release|"
     }
 
+.. _http-restful-api-artifact-set-properties:
+
+Set Artifact Properties
+=======================
+To set properties for a specific version of an artifact, submit an HTTP PUT request::
+
+  PUT <base-url>/namespaces/<namespace>/artifacts/<artifact-name>/versions/<artifact-version>/properties
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``<namespace>``
+     - Namespace ID
+   * - ``<artifact-name>``
+     - Name of the artifact.
+   * - ``<artifact-version>``
+     - Version of the artifact.
+
+The request body must be a JSON object that contains the properties for the artifact.
+The keys and values in the object must be strings. If any properties are already
+defined, they will be overwritten.
+
+.. container:: highlight
+
+  .. parsed-literal::
+    |$| PUT <base-url>/namespaces/default/artifact/WordCount/versions/|release| -d 
+    {
+        "author": "samuel",
+        "company": "cask"
+    }
+
+.. _http-restful-api-artifact-set-property:
+
+Set Artifact Property
```

Comment:
We are also inconsistent with respect to titles. But I think we should probably add "a" or "a" to these singular titles:
```
Set an Artifact Property
Retrieve an Artifact Property
Delete an Artifact Property

Of course, adjusting the underlines. There are a few other singular titles in this file that could use this:

Retrieve an Artifact Detail
Delete a System Artifact